Index: Assets/Player/Mechanics/Cannon/CannonWeapon.cs.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/Player/Mechanics/Cannon/CannonWeapon.cs.meta b/Assets/Player/Mechanics/Cannon/CannonWeapon.cs.meta
new file mode 100644
--- /dev/null	(date 1738356903633)
+++ b/Assets/Player/Mechanics/Cannon/CannonWeapon.cs.meta	(date 1738356903633)
@@ -0,0 +1,3 @@
+fileFormatVersion: 2
+guid: b8ce213226224aba817a1279ef2ca073
+timeCreated: 1738356903
\ No newline at end of file
Index: Assets/Player/Mechanics/Cannon/CannonWeapon.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/Player/Mechanics/Cannon/CannonWeapon.cs b/Assets/Player/Mechanics/Cannon/CannonWeapon.cs
new file mode 100644
--- /dev/null	(date 1738356954692)
+++ b/Assets/Player/Mechanics/Cannon/CannonWeapon.cs	(date 1738356954692)
@@ -0,0 +1,5 @@
+
+public class CannonWeapon
+{
+    
+}
Index: Assets/Player/Mechanics/Laser/LaserWeapon.cs.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/Player/Mechanics/Laser/LaserWeapon.cs.meta b/Assets/Player/Mechanics/Laser/LaserWeapon.cs.meta
new file mode 100644
--- /dev/null	(date 1738356919155)
+++ b/Assets/Player/Mechanics/Laser/LaserWeapon.cs.meta	(date 1738356919155)
@@ -0,0 +1,3 @@
+fileFormatVersion: 2
+guid: 1e66341a6222407a8fc7a3379ee0e4b0
+timeCreated: 1738356919
\ No newline at end of file
Index: Assets/Player/Mechanics/Laser/LaserWeapon.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/Player/Mechanics/Laser/LaserWeapon.cs b/Assets/Player/Mechanics/Laser/LaserWeapon.cs
new file mode 100644
--- /dev/null	(date 1738356925026)
+++ b/Assets/Player/Mechanics/Laser/LaserWeapon.cs	(date 1738356925026)
@@ -0,0 +1,5 @@
+
+public class LaserWeapon
+{
+    
+}
Index: Assets/Player/Mechanics/Sword/SwordWeapon.cs.meta
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/Player/Mechanics/Sword/SwordWeapon.cs.meta b/Assets/Player/Mechanics/Sword/SwordWeapon.cs.meta
new file mode 100644
--- /dev/null	(date 1738356931183)
+++ b/Assets/Player/Mechanics/Sword/SwordWeapon.cs.meta	(date 1738356931183)
@@ -0,0 +1,3 @@
+fileFormatVersion: 2
+guid: f9a1eca6c9334680968b9dd908bbe2fb
+timeCreated: 1738356931
\ No newline at end of file
Index: Assets/Player/Mechanics/Sword/SwordWeapon.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/Player/Mechanics/Sword/SwordWeapon.cs b/Assets/Player/Mechanics/Sword/SwordWeapon.cs
new file mode 100644
--- /dev/null	(date 1738356936389)
+++ b/Assets/Player/Mechanics/Sword/SwordWeapon.cs	(date 1738356936389)
@@ -0,0 +1,5 @@
+
+public class SwordWeapon
+{
+    
+}
